### PR TITLE
Fix httpd and update README (SNO)

### PR DESCRIPTION
**Pull request by vitus133**
Hey, I added the httpd setup to SNO deploy. Had some time to update the readme while it was installing :)
@serngawy @yrobla

### CODE REVIEW

File: ai-deploy-cluster-singlenode/README.md [hunk 1]
```diff
@@ -1,125 +1,135 @@
...
+|                     Host                     | Cluster  |                  Id                  |   Status  |  Role  |       Ip       |
++----------------------------------------------+----------+--------------------------------------+-----------+--------+----------------+
+| single-node-cluster.test-aut.cluster.testing | test-aut | 0471f8d5-812a-1e08-42e0-27e173750188 | installed | master | 192.168.112.34 |
++----------------------------------------------+----------+--------------------------------------+-----------+--------+----------------+
+```
+
+Now you can download the kubeconfig of your cluster using the command below:
+
+```console
+$ aicli -U http://localhost:8080 download kubeconfig {YOUR_CLUSTER_NAME} --path ~/
```
Comment (yrobla) [inline ai-deploy-cluster-singlenode/README.md:127]: maybe not just localhost , but specify that you can declara a var AICLI_URL, that needs to point to the ip of your assisted installer , and that needs to match the one in the inventory.

Comment (vitus133) [reply ai-deploy-cluster-singlenode/README.md:127]: OK, fixing that

Comment (vitus133) [reply ai-deploy-cluster-singlenode/README.md]: Done

**Comment by vitus133** [comment]
squashed

**Review by yrobla — APPROVED** [review]

**Comment by vitus133** [comment]
Thanks!

**Pull request merged**